webui/src/pages/repositories/repository/compare.jsx

```diff
@@ -144,24 +161,35 @@ const CompareList = ({ repo, reference, compareReference, prefix, onSelectRef, o
                                             rightCommittedRef += "@";
                                         }
                                         return (
-                                            <TreeItem key={entry.path + "-item"} entry={entry} repo={repo}
-                                                      reference={reference}
-                                                      internalReferesh={internalRefresh} leftDiffRefID={leftCommittedRef}
-                                                      rightDiffRefID={rightCommittedRef} delimiter={delimiter}
-                                                      relativeTo={prefix}
-                                                      onNavigate={onNavigate}
-                                                      getMore={(afterUpdatedChild, path, useDelimiter = true, amount = -1) => {
-                                                          return refs.diff(repo.id, reference.id, compareReference.id, afterUpdatedChild, path, useDelimiter ? delimiter : "", amount > 0 ? amount : undefined);
-                                                      }}/>);
+                                            showDeltaDiffCompareView
```

Comment:
This `else` is getting very long-winded. I'd use `{}` instead of `()` for the `else` and break this down into a few more steps instead of trying to nest everything. Loops within ternary ifs within ternary ifs. It's getting to be unreadable IMHO